### ISSUE TITLE
Collection page tabs should have aria-current

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1757]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

Collection page, e.g. https://findingaids.princeton.edu/catalog/C0178

### Description of error / warning

When navigating the page with VoiceOver it doesn't announce when it's reading the current tab

### WCAG success criterion

[4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG22/#name-role-value)

"ARIA is used appropriately to enhance accessibility when HTML is not sufficient."

### WCAG conformance level

A

### Recommended fix

Add an aria-current to the selected tab

### Tool used to find the problem

VoiceOver
